rs/tests/ckbtc/ckbtc_minter_basics_test.rs

```diff
@@ -1,3 +1,111 @@
+#[rustfmt::skip]
```

Comment:
Is this still needed? In my inlining efforts I have been removing these rustfmt::skip statements.